src/commands/run-command.yml

```diff
@@ -1,21 +1,22 @@
 description: >
-  Execute a MATLAB script, function, or statement. MATLAB exits with exit code 0 if the specified
-  script, function, or statement executes successfully without error. Otherwise, MATLAB terminates
-  with a nonzero exit code, which causes the build to fail. You can use the assert or error
-  functions in the command to ensure that builds fail when necessary. When you use this command,
-  all of the required files must be on the MATLAB search path.
```

Comment:
Is it fine that we no longer have the last sentence about required files?

Reply:
What do you think?

Reply:
It could be helpful to keep, I guess? (Could be a pitfall for some users.)